index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Library App</title>
+        <link rel="stylesheet" href="assets/style/style.css" type="text/css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="title"> Library App</div>
+
+            <div class="formcontainer">
+                <form id="bookForm" class="form" > 
+                    <label > title</label>
+                    <input id="title" type="text" name="booktitle" class="inputs">
+
+                    <label > author</label>
+                    <input id="author" type="text" name="bookauthor" class="inputs">
+
+                    <label > pages</label>
+                    <input id="pages" type="text" name="bookpages" class="inputs">
+
+                    <label for="read">Check if you read this book</label>
+                    <input id="read" type="checkbox" name="read" class="">
+                    
+                    <input id="submitForm" type="button" value="submit" class="buttons">
+                </form>
+            </div>
+
```

Comment:
### [OPTIONAL]

Kindly add validations to your project. This will ensure that a person with bad intentions cannot crush your application. It is usually a good practice to add validation to your application. You can do this by using HTML5 require attribute. Kindly read more on this [here](https://www.w3schools.com/tags/att_input_required.asp). Also for your number of pages, you can use a number type field instead of a text field